src/Foundation/EnumDesktop.cs

```diff
@@ -96,6 +96,7 @@ public enum AEEventClass : uint {
 
 	// Added from NSUserNotification.h
 	[Native]
+	[Advice ("'NSUserNotification' usages should be replaced with 'UserNotifications framework'.")]
```

Comment:
You put the ' at the wrong place on this one (compared the your other advice). It's 'UserNotifications'.